charts/rancher-monitoring/v0.2.0/charts/alertmanager/templates/alertmanager.yaml

```diff
@@ -45,6 +45,7 @@ spec:
 {{- end }}
 {{- if or .Values.storageSpec .Values.persistence.enabled }}
   storage:
+    disableMountSubPath: {{ .Values.disableMountSubPath }}
```

Comment:
Unsure if this is necessary since I'm not aware of a way to enable persistent volumes in any way for the Alertmanager deployed by Alerting V1 since there's no `Answers` that can be supplied.

Should this be omitted or just kept in for good measure?

Reply:
Not really sure about this flag; but it might be safer to keep it around